store/store-connection-manager/src/createNewStoreController.ts

```diff
@@ -60,7 +62,7 @@ export async function createNewStoreController (
     cacheDir: opts.cacheDir,
     cert: opts.cert,
     fullMetadata,
-    filterMetadata: fullMetadata,
+    filterMetadata,
```

Comment:
you should filter metadata always. Just update the code that filters to keep the libc field.

Reply:
It does mean that the bug won't immediately be fixed unless the user clears the old metadata cache.

Reply:
you can bump the version in FULL_FILTERED_META_DIR